src/dev/build/lib/version_info.ts

```diff
@@ -28,7 +28,10 @@ export async function getVersionInfo({ isRelease, versionQualifier, pkg }: Optio
   );
 
   return {
-    buildSha: (await execa('git', ['rev-parse', 'HEAD'])).stdout,
+    buildSha:
+      process.env.GIT_COMMIT ||
+      process.env.BUILDKITE_COMMIT ||
+      (await execa('git', ['rev-parse', 'HEAD'])).stdout,
```

Comment:
does this need the .git folder check too?

Reply:
`BUILDKITE_COMMIT` will always be set, so it shouldn't need it